### ISSUE TITLE
how can i know when it finish update task?

### ISSUE THREAD

**Issue by fengmao31** [jstaf/onedriver#112]
![image](https://user-images.githubusercontent.com/19343842/110787108-ecdd2f80-82a7-11eb-9736-a9ebd34685b0.png)
I shut down the pc. then I use my laptop to watch my homework. I find it is 0 bit.
when I use pc again. The 0-bit file rewrites my homework.
luckily I print it to pdf and update it to moodle.

**Comment by jstaf** [comment]
This is the same issue as #99, I'm going to close this as a duplicate.

**Comment by fengmao31** [comment]
no，no，no, I want to know when it finish the task about updating. i know the bug.
I want to the gui have a state to tell the user if it is ok.





------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Jeff ***@***.***&gt;; 
发送时间: 2021年3月16日(星期二) 中午11:49
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [jstaf/onedriver] how can i know when it finish update task? (#112)





 
This is the same issue as #99, I'm going to close this as a duplicate.

**Comment by jstaf** [comment]
Ah, like this? https://support.microsoft.com/en-us/office/what-do-the-onedrive-icons-mean-11143026-8000-44f8-aaa9-67c985aa49b3

**Comment by fengmao31** [comment]
only need to know the state of update . I think the 0-byte file is because people don't give enough time to update the file.
by the way, can the one driver go through pac VPN?

**Comment by jstaf** [comment]
Fair enough, I'll reopen this with the goal of having some sort of visual indicator around uploads. (Admittedly I'm not too sure how to implement this because it's highly dependent on what desktop environment you use and most DEs just think files from onedriver are normal files.)

onedriver should be able to go through any VPN, but it's up to you to configure the networking. onedriver just uses whatever the default routing is for your computer so as long as your VPN changes the default network routing to send traffic through it, onedrive traffic should go through the VPN (most VPNs will probably do the right thing by default).

**Comment by rodrigost23** [comment]
@jstaf The official Nextcloud client has those indicators, at least for Nautilus (I haven't tried it in other file managers). Maybe that can serve as inspiration?  
  
[https://github.com/nextcloud/desktop/tree/master/shell_integration](https://github.com/nextcloud/desktop/tree/master/shell_integration)

**Comment by almusx** [comment]
> Fair enough, I'll reopen this with the goal of having some sort of visual indicator around uploads. (Admittedly I'm not too sure how to implement this because it's highly dependent on what desktop environment you use and most DEs just think files from onedriver are normal files.)
> 
> onedriver should be able to go through any VPN, but it's up to you to configure the networking. onedriver just uses whatever the default routing is for your computer so as long as your VPN changes the default network routing to send traffic through it, onedrive traffic should go through the VPN (most VPNs will probably do the right thing by default).

Can you change the file extension to something like .partial like some browsers do when downloading a file?
